frontend/src/assets/css/styles.scss

```diff
@@ -1 +1,4 @@
 @import "./scaffolding.scss";
+@import "./fonts.scss";
+@import "./vars.scss";
+@import "./typography.scss";
```

Comment:
something with line ending here I guess

Reply:
Yes, I know. I used styleling --fix to fix property sorting, I think that's the reason.